freshmaker/parsers/errata/state_change.py

```diff
@@ -40,13 +40,17 @@ def parse(self, topic, msg):
         msg_id = msg.get('msg_id')
         inner_msg = msg.get('msg')
         errata_id = int(inner_msg.get('errata_id'))
+        new_state = inner_msg.get('to')
+
+        # Ingore other state change messages which are not for SHIPPED_LIVE
+        if new_state != "SHIPPED_LIVE":
```

Comment:
Just asking, on *rebuild_images_on_rpm_advisory_change.py:107* we have condition `if event.advisory.state != 'SHIPPED_LIVE'` and based on it we will generate composes. If I understood right now it will not make sense, because we are reacting only on advisories that are already in 'SHIPPED_LIVE' state?

Reply:
`rebuild_images_on_rpm_advisory_change.py` handles `ErrataAdvisoryRPMsSignedEvent` events, but such type events can come from:

1. all rpms are signed in an advisory, then an `ErrataAdvisoryRPMsSignedEvent` is generated.
2. advisory state is moved to `SHIPPED_LIVE`, then a faked `ErrataAdvisoryRPMsSignedEvent` is generated.

Case 2 is what we're changing in this PR, and we don't need to check advisory state again. But in case 1, an advisory can be in other states other than `SHIPPED_LIVE`, so we still need to check advisory state.

Seems this `ErrataAdvisoryRPMsSignedEvent` exists for some historical reason and probably we can clean up and simply the code to have one event (e.g. `RPMAdvisoryShippedEvent`) generated directly from advisory state change message and remove the code of handling rpm signed messages. I'd recommend having a look on the code base and check whether this is true.